### DISCUSSION TITLE
Failed to start milvus

### DISCUSSION THREAD

**Discussion by wangsj2021** [milvus-io/milvus#27096] [Q&A and General discussion]
(base) [root@localhost milvus]# sudo docker compose up -d
[+] Running 2/3
 ✔ Container milvus-minio       Running                                                                                                                                                                                     0.0s 
 ✔ Container milvus-etcd        Running                                                                                                                                                                                     0.0s 
 ⠿ Container milvus-standalone  Starting                                                                                                                                                                                    0.6s 
Error response from daemon: could not select device driver "matrox" with capabilities: [[gpu]]

**Comment by xiaofan-luan** [comment]
is this gpu instance? could you offer us detailed logs?

**Comment by yhmo** [comment]
Seems it cannot be compatible with the Matrox device. Is your matrox C-series or D-series?
C-series is based on AMD GPU, not support currently.
D-series is based on NVIDIA Quadro. For milvus, the prerequisites for GPU: https://milvus.io/docs/install_standalone-gpu-docker.md#Prerequisites. The required compute capability of GPU device is 6.1, 7.0, 7.5, or 8.0. You can check the capability of your GPU in this page: https://developer.nvidia.com/cuda-gpus

**Reply by labroskokkalas** [reply]
I have a RTX A5000 with compute capability 8.6 and I have the same issue. Will 8.6 be supported in the future?

**Reply by yhmo** [reply]
> I have a RTX A5000 with compute capability 8.6 and I have the same issue. Will 8.6 be supported in the future?

Yes, I believe.

**Comment by WindyOrgUK** [comment]
Not intending to steal the topic but I see the same when running 2.3.3 with 2 NVIDIA GPUS (RTX 3660 and RTX1660)

Error response from daemon: could not select device driver "nvidia" with capabilities: [[gpu]]

my nvidia-smi output:

`nvidia-smi --query-gpu=compute_cap --format=csv'
'compute_cap
8.6
7.5`

======================
 NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2

I tried all the combinations of single and multiple GPUs in "device_ids:" 
I was told on Discord channel that v2.3.3 suppose to support compute_cap 8.6

**Comment by presburger** [comment]
@WindyOrgUK @wangsj2021 I apologize for any inconvenience or negative experience I may have caused you. Please try install nvidia container toolkit, follow this, [nvidia container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)

**Comment by WindyOrgUK** [comment]
Thank you @Presburger, this seems to have worked (at least no error on start)
Testing now.

**Comment by WindyOrgUK** [comment]
So far so good,
I am ingesting embeddings of Arxiv docs (https://towardsdatascience.com/arxiv-scientific-papers-vector-similarity-search-15059970a73a)
and can confirm my RTX 3060 (compute_cap = 8.6) is being utilised.
Thank you @Presburger for the solution tip.

**Comment by WindyOrgUK** [comment]
Just one more question.
Has this been tested on multi-GPU systems ?
I t seems that regardless of the "device_ids:" settings in the .yml file, always ID:0 is used

`   deploy:
      resources:
        reservations:
          devices:
            - driver: nvidia
              capabilities: ["gpu"]
              device_ids: ["1"]`
